example-app/SantokuApp/jest/__mocks__/@notifee/react-native.ts

```diff
@@ -0,0 +1,69 @@
+import notifee from '@notifee/react-native';
+export * from '@notifee/react-native/dist/types/Library';
+export * from '@notifee/react-native/dist/types/Notification';
+export * from '@notifee/react-native/dist/types/Trigger';
+export * from '@notifee/react-native/dist/types/NotificationIOS';
+export * from '@notifee/react-native/dist/types/NotificationAndroid';
+
+const mock: typeof notifee = {
```

Comment:
Did this not work? (In jest setup)
```
jest.mock('@notifee/react-native', () => require('@notifee/react-native/jest-mock'))
```
https://github.com/invertase/notifee/tree/main/packages/react-native#jest-testing

Reply:
Yes.
Some interfaces, such as createChannels, seem to be missing.

Reply:
hmmm...